dataprofiler/tests/profilers/test_column_profile_compilers.py

```diff
@@ -281,10 +281,9 @@ def test_base(self, *mocks):
                                 'o': 2, 's': 2, 't': 2, 'y': 1},
                 'vocab': [' ', '-', '.', '1', '2', '3', '4', 'D', 'J', 'a', 'e',
                           'h', 'i', 'm', 'n', 'o', 's', 't', 'y'],
-                'word_count': {'123': 1, '1234': 1, '432': 1, 'Doe': 1,
+                'word_count': {'123-432-1234': 1, 'Doe': 1,
```

Comment:
Is this correction related to a code change in this PR or is this just a separate correction?

Reply:
I discussed with Grant on this, and we agree on this expected output of the word count. So, i changed it here.